workitems/docs/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.4.3 - 2024-02-06
+
+- Add exception property to workitems
+
```

Comment:
This date is good, but if you plan to make the release tomorrow, you should update it:
```suggestion
## 1.4.3 - 2024-02-07

- Add exception property to input work items.

```

But if you release today, all good then.